### ISSUE TITLE
readme: Add example to install kubeseal on linux

### ISSUE THREAD

**Issue by sebastian-philipp** [bitnami-labs/sealed-secrets#907]
Would be great to also have an example how how to easily install the binary on linux, not just for mac. 

I ended up with 

```
curl -s https://api.github.com/repos/bitnami-labs/sealed-secrets/releases/latest | jq -r '.assets[] | select(.name | endswith("linux-amd64.tar.gz")) | .browser_download_url' | wget -qi -
tar -xvzf kubeseal-*-linux-amd64.tar.gz kubeseal
mkdir -p ~/.local/bin
mv ./kubeseal ~/.local/bin/kubeseal
```

Would be great to have something like this in the readme.

**Comment by alvneiayu** [comment]
hi @sebastian-philipp 

First of all, thanks a lot for sharing your solution. We are reviewing already a PR including a possible solution [here](https://github.com/bitnami-labs/sealed-secrets/pull/913).

Thanks

Álvaro

**Comment by alemorcuq** [comment]
Solved in #913
